tests/utils/test_notifications.py

```diff
@@ -353,6 +353,9 @@ def test_inverse_dependencies(
     )
 
     custom_notification_pusher.push("test-message", "info", [])
+    import time
+
+    time.sleep(3)
```

Comment:
mind adding some retry until succesfull with a 10 second timeout ? i think it will fail transiently

Reply:
Ah this is a leftover. removing it

Reply:
Ah this is a leftover, isn't needed. removing